src/components/DeploymentsList.vue

```diff
@@ -239,8 +239,10 @@
 }
 
 .deployment-list__name { @apply
+  inline-flex
```

Comment:
idk if it wrapped well before but does making this flex cause any layout issues where the name won't wrap when it should?

Reply:
I don't feel strongly. Looks better inline vs flex I think but up to you. Just mostly wanted to make sure the name itself still wrapped.

Reply:
Hmm took another stab and needs some more tinker time to get inlining space _just right_ but changes as-is should improve the majority of cases i think and i'll maybe revisit styling here later.

Reply:
Sounds good 👍 Ty!